community/ecosystem-projects.md

```diff
@@ -0,0 +1,49 @@
+# Ecosystem Projects
+
+The Argoproj consists of four core projects and set of supporting ecosystem projects. The ecosystem projects implement
+experimental features or opinionated use cases that are useful for the sub-set of users.
+
+## Eligibility
+
+The ecosystem project must enhance one of the existing core projects as opposed to focusing on a totally new area not related to Argo.
+
```

Comment:
Although we don't have any **_officially_** supported client libraries (e.g. a workflow java/python client library), where would something like an official java client library be categorized?

Reply:
I think an officially supported client library is an ecosystem project, hosted in https://github.com/argoproj